package.json

```diff
@@ -24,8 +24,9 @@
     "jsdoctest": "^1.7.1",
     "mocha": "^4.0.0",
     "random-js": "^1.0.4",
-    "rollup": "^0.51.0",
-    "rollup-plugin-uglify": "^2.0.1",
+    "rollup": "*",
```

Comment:
I'd rather keep at least a version range here, relying on 'latest' is bound to break in the nearish future?

Reply:
Sounds like a plan!

Reply:
👍 Updated devDependencies